registrations/tasks.py

```diff
@@ -269,6 +270,40 @@ def create_subscriptionrequests(self, registration):
         }
         SubscriptionRequest.objects.create(**mother_sub)
 
+        # Send registration welcome SMS
+        if registration.data["msg_receiver"] == "mother_to_be":
+            if "hw" in registration.source.authority:
+                sms = settings.MOTHER_HW_WELCOME_TEXT_UG_ENG
+            else:
+                sms = settings.MOTHER_PUBLIC_WELCOME_TEXT_UG_ENG
+
+            # Insert the mother's name in the SMS
+            sms = sms.replace(
+                '[mother_first_name]', registration.data["mama_name"])
+        else:
+            if "hw" in registration.source.authority:
+                sms = settings.HOUSEHOLD_HW_WELCOME_TEXT_UG_ENG
+            else:
+                sms = settings.HOUSEHOLD_PUBLIC_WELCOME_TEXT_UG_ENG
+        # Insert the mother's health_id in the SMS
+        mother = utils.get_identity(registration.mother_id)
+        if mother["details"]["health_id"]:
+            sms = sms.replace(
+                '[health_id]', str(mother["details"]["health_id"]))
+        else:
+            # TODO: raise exception?
```

Comment:
Please open an Issue for this. This is part of the bigger narrative of "what if the ID generator fails".